Source/Scene/Cesium3DTileStyleEngine.js

```diff
@@ -136,6 +138,10 @@ define([
             feature.backgroundYPadding = style.backgroundYPadding.evaluate(frameState, feature);
             feature.backgroundEnabled = style.backgroundEnabled.evaluate(frameState, feature);
 
+            // Create a color material with fromType:
+            feature.material = Material.fromType('Color');
```

Comment:
With the `anchorLineColor` property suggestion above, this would become:
```javascript
if (defined(feature.anchorLineColor)) {
    feature.anchorLineColor = style.anchorLineColor.evaluateColor(frameState, feature);
}
```